Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift

```diff
@@ -135,6 +106,51 @@ struct DashboardView: View {
         }
     }
     
+    private var maintenanceListView: some View {
```

Comment:
Also I think for right now, it's fine to leave this where it was, and not make a variable of it, just yet